vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/textfield/TextFieldTester.java

```diff
@@ -24,7 +24,7 @@
  */
 @Tests({ TextField.class, PasswordField.class, EmailField.class,
         BigDecimalField.class })
-public class TextFieldTester<T extends InternalFieldBase<T, V>, V>
+public class TextFieldTester<T extends TextFieldBase<T, V>, V>
```

Comment:
class import is missing?

Reply:
interesting.. locally it builds for me.. aww.

Reply:
oh, it failed on other part.. for this line, it doesn't need extra class import

```
/opt/agent/work/ef9ade89d4c3cefd/vaadin-testbench-unit-junit5/src/test/java/com/vaadin/testbench/unit/ComponentQueryTest.java:[28,43] cannot find symbol
[11:30:05]	[Step 4/5] [ERROR]   symbol:   class InternalFieldBase
[11:30:05]	[Step 4/5] [ERROR]   location: package com.vaadin.flow.component.textfield
[11:30:05]	[Step 4/5] [ERROR] /opt/agent/work/ef9ade89d4c3cefd/vaadin-testbench-unit-junit5/src/test/java/com/vaadin/testbench/unit/ComponentQueryTest.java:[496,19] cannot find symbol
[11:30:05]	[Step 4/5] [ERROR]   symbol:   class InternalFieldBase
[11:30:05]	[Step 4/5] [ERROR]   location: class com.vaadin.testbench.unit.ComponentQueryTest
[11:30:05]	[Step 4/5] [ERROR] -> [Help 1]
```